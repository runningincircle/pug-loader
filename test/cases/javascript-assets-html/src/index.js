const html = require('Views/widget.pug');
console.log(html);