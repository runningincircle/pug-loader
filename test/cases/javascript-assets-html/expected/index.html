<h1>Hello!</h1><p>../images/image.png</p><img src="assets/images/image.697ef306.png"><p>Images/image.png</p><img src="assets/images/image.697ef306.png"><p>&quot;Images/&quot; + file2</p><img src="assets/images/image2.697ef306.png"><p>`Images/${file2}`</p><img src="assets/images/image2.697ef306.png"><p>Hello green (#0f0)!</p><div class="first"><i>red</i><i>green</i><i>blue</i></div><div class="second"><h3>Red Green Blue</h3><i>red</i><i>green</i><i>blue</i><h3>Rose Violet Azure</h3><i>rose</i><i>violet</i><i>azure</i></div>